### PR TITLE
EOS-13759 [LDR-R1 SW Upgrade] for s3backgrounddelete

### PR DESCRIPTION
**Pull request by shanbhagsv1**

### CODE REVIEW

File: s3backgrounddelete/s3backgrounddelete/config/s3_background_delete_config.yaml [hunk 1]
```diff
@@ -17,6 +17,9 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+version_config:                             # Section for version of S3 background delete config
+   version: 1.0.0                                          # Default version for S3 background delete config
```
Comment (abhilekh268) [inline s3backgrounddelete/s3backgrounddelete/config/s3_background_delete_config.yaml:21]: keep it 1 only.as per guidelines.

**Review by abhilekh268 — APPROVED** [review]

File: s3backgrounddelete/s3backgrounddelete/config/s3_background_delete_config.yaml (hunk 1 above)
Comment (shanbhagsv1) [reply s3backgrounddelete/s3backgrounddelete/config/s3_background_delete_config.yaml]: Done

**Comment by shanbhagsv1** [comment]
Build Job http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/S3server/49/

File: s3backgrounddelete/s3backgrounddelete/config/s3_background_delete_config.yaml [hunk 2]
```diff
@@ -17,6 +17,9 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+version_config:                             # Section for version of S3 background delete config
+   version: 1                                              # Default version for S3 background delete config
```
Comment (amitwac2608) [inline s3backgrounddelete/s3backgrounddelete/config/s3_background_delete_config.yaml:21]: I have always observed that version is always a 3 dot value, like 1.0.0.

Can we rethink on why to stick to single digit version ? @nileshgovande @bkirunge7 !

File: s3backgrounddelete/s3backgrounddelete/cortx_s3_config.py [hunk 3]
```diff
@@ -70,6 +70,13 @@ def _load_and_fetch_config(self):
     def get_s3recovery_flag(self):
         return self.s3recovery_flag
 
+    def get_config_version(self):
+        """Return version of S3 background delete config file or KeyError."""
```
Comment (amitwac2608) [inline s3backgrounddelete/s3backgrounddelete/cortx_s3_config.py:74]: is it the version of the config file or the backgrounddelete service ?

**Review by amitwac2608 — COMMENTED** [review]
commented a couple of questions. Rest is good.

**Review by nileshgovande — APPROVED** [review]

**Pull request merged**